### PR TITLE
Prepare multiple openapi entrypoints to replace with a single one

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

**Review by adamaltman — DISMISSED** [review]
Redocly is changing the `.redocly.yaml` config format. This will enable some optimization later on.

File: plugins/products-bundler/mapping/Core.yaml [hunk 1]
```diff
@@ -34,6 +37,10 @@ x-tagGroups:
       - Custom Fields
       - Files
       - Search
+      - Gateway Accounts
```
Comment (ashkarpetin) [inline plugins/products-bundler/mapping/Core.yaml:40]: Should not be in a core

Comment (VladimirBerdnik) [reply plugins/products-bundler/mapping/Core.yaml:40]: It is required to include x-webhooks with this tag. Only webhooks are included without operations

File: plugins/products-bundler/mapping/Core.yaml [hunk 2]
```diff
@@ -34,6 +37,10 @@ x-tagGroups:
...
+      - Application owners
+      - Application users
+      - Balance transactions
```
Comment (ashkarpetin) [inline plugins/products-bundler/mapping/Core.yaml:43]: All of these are in the Users

Comment (VladimirBerdnik) [reply plugins/products-bundler/mapping/Core.yaml:43]: These are required to include x-webhooks, no operations will be added
image

Comment (ashkarpetin) [reply plugins/products-bundler/mapping/Core.yaml:43]: These webhooks should be in users

Comment (VladimirBerdnik) [reply plugins/products-bundler/mapping/Core.yaml:43]: I thought all webhooks are located in Core

Comment (VladimirBerdnik) [reply plugins/products-bundler/mapping/Core.yaml:43]: I see 5 webhooks in Users, maybe they were added by mistake

Comment (VladimirBerdnik) [reply plugins/products-bundler/mapping/Core.yaml]: Adjusted tags and webhooks, thank you!

File: openapi/core.yaml [hunk 3]
```diff
@@ -197,12 +206,15 @@ x-tagGroups:
   - name: Payments
     tags:
       - Transactions
+      - Transactions Timeline
```
Comment (ashkarpetin) [inline openapi/core.yaml:209]: Can we move timelines at the end of the list until we will be able to move them to Users?

Comment (VladimirBerdnik) [reply openapi/core.yaml]: Moved! :smile:

**Review by adamaltman — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**